src/drivers/github_test.go

```diff
@@ -114,31 +118,17 @@ func TestGitHubDriver_MergePullRequest_PullRequestNotFound(t *testing.T) {
 	httpmock.RegisterResponder("GET", currentPullRequestURL, httpmock.NewStringResponder(200, "[]"))
 	_, err := driver.MergePullRequest(options)
 	assert.Error(t, err)
-	assert.Equal(t, "no pull request found", err.Error())
-}
-
-func TestGitHubDriver_MergePullRequest_MultiplePullRequestsFound(t *testing.T) {
```

Comment:
This test checks a condition that can never happen, since we aren't calling `driver.MergePullRequest` when there are multiple pull requests anymore.